utils/scripts/split-graph-data.jl

```diff
@@ -0,0 +1,66 @@
+using DataFrames
+
+include("../csv-modifications.jl")
+
+# Local cleanup to ensure clean files
+apply_to_files_named(rm, "graph-assets-data.csv")
+apply_to_files_named(rm, "graph-flows-data.csv")
+run(`git restore test/inputs/ benchmark/EU/`)
+
+# graph-assets-data.csv is created by simplifying assets-data.csv (and flows)
+apply_to_files_named("assets-data.csv") do path
+    # Old file changes
+    change_file(path) do old_tcsv
+        new_file = "graph-assets-data.csv"
+        new_file_path = joinpath(dirname(path), new_file)
+
+        static_cols = [:type, :group]
+
+        # New file changes
+        touch(new_file_path)
+        change_file(new_file_path) do tcsv
+            cols = [:name; static_cols]
+            indexes = columnindex.(Ref(old_tcsv.csv), cols)
+
+            tcsv.units = old_tcsv.units[indexes]
+            tcsv.csv = unique(old_tcsv.csv[:, indexes])
+
+            idx = findfirst(names(tcsv.csv) .== "investment_cost")
+            add_column(tcsv, "investment_method", "simple"; unit = "{simple,compact}")
```

Comment:
```suggestion
            add_column(tcsv, "investment_method", "simple"; unit = "{simple;compact}")
```
I think using ; is more consistent with the rest?

Reply:
Mistake on my part, should have been a `;`.